src/main/java/org/apache/fineract/api/BatchApi.java

```diff
@@ -301,4 +304,20 @@ private String createDetailsFile(List<Transfer> transfers) {
         return null;
     }
 
+    private BatchDTO generateBatchSummaryResponse(Batch batch) {
+        double batchFailedPercent = 0;
+        double batchCompletedPercent = 0;
+
+        if(batch.getTotalTransactions() != null){
+            batchFailedPercent = (double) batch.getFailed() / batch.getTotalTransactions() * 100;
```

Comment:
this may not give the perncetage in double, i suspect this may give decimal points as 00 always
suggetions:
1. use 100.0 
2. do roundoff to get only 2 decimal points